src/SarifCli/AnalyzeOptions.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CommandLine;
+using Microsoft.CodeAnalysis.Sarif.Driver;
+
+namespace Microsoft.CodeAnalysis.Sarif.Cli
+{
+    [Verb("validate", HelpText = "Validate a SARIF file against the schema and against additional correctness rules.")]
```

Comment:
It's not the case that there's only one arg. Remember that this derives from `AnalyzeOptionsBase`, from which I get (among others) `--output`. Most important, `AnalyzeOptionsBase` gives me the unnamed argument list `TargetFileSpecifiers` -- the input files -- so I can't use an unnamed arg for the schema.

I will, however, change the long-form argument name to `json-schema`. I won't make it just `schema` because that wouldn't match the short form `j` (and I can't use `s`  for the short form because it's already used for `statistics`).

---
In reply to: [77238716](https://github.com/Microsoft/sarif-sdk/pull/501#discussion_r77238716) [](ancestors = 77238716)